css/sytles.css

```diff
@@ -0,0 +1,58 @@
+div.container {
```

Comment:
@ermajinkles this could have been written as `.container { ... }`, div .container is not necessary.